Application/src/view/visuals/Draw.java

```diff
@@ -191,6 +213,17 @@ private void init(Canvas canvas) {
     }
 
     /**
+     * Initializes the drawing of class diagram
+     * @param gc
+     */
+    void initClassDiagram(GraphicsContext gc){
+        gc.clearRect(0,0,getWidth(), getHeight());
+        // adds the background to class diagram canvas
+        gc.drawImage(classDiagramBackground,0,0, this.canvas_class.getWidth(), this.canvas_class.getHeight());
+    }
+
+    private void clear(GraphicsContext gc) {
+        gc.clearRect(0,0,getWidth(), getHeight()); // Clears the canvas
      * Clear the provided GraphicalContext.
```

Comment:
typo; missing bracket and /* to start the java doc comment

Reply:
not part of this pull request

Reply:
@IsabelleTornqvist  you added a clear method on top of the old one, now there are 2 clear methods and a compilation error